src/odemis/util/transform.py

```diff
@@ -818,3 +818,176 @@ def inverse(self):
         Ainv = numpy.dot(tri_inv(SL), numpy.transpose(self.rotation_matrix))
         tinv = -numpy.dot(Ainv, self.translation)
         return self.__class__(matrix=Ainv, translation=tinv)
+
+
+class AnamorphosisTransform(AffineTransform):
+    """
+    AnamorphosisTransform
+
+    The anamorphosis transform is a polynomial model of the distortions in an
+    electron optical system, and is described in more detail in [1].
+
+    The anamorphosis transform has the following form:
+
+        wᵢ = A + B₁ w + B₂ w̄ + C₁ w² + C₂ ww̄ + C₃ w̄² +
+                                       D₁ w²w̄ + D₂ ww̄² + E₁ w³w̄² + E₂ w²w̄³,
+
+    where A, B₁, B₂, C₁, C₂, C₃, D₁, D₂, E₁, and E₂ are the (complex)
+    coefficients of the transform. The in- and output coordinates are the
+    complex numbers w = x + j*y. Note that A, B₁, B₂ define an ordinary affine
```

Comment:
It would be good to note here that `w̄ ` is the complex conjugate of `w`

Reply:
Added note to docstring